.github/CONTRIBUTING.md

```diff
@@ -1,93 +1,146 @@
+
+Keep in mind that the PR should be named in reference of the main fix or feature you provide, minor information can be added in the description.
+
+Bad
+
+> Fix errors in installation method, update dependencies and improve installation documentation
+
+Good
 
-### Type
+> Fix installation method
 
-Must be one of the following:
+> What's the new behaviour?
+> 
+> * Dependencies updated
+> * Documentation improved
 
-* **feat**: A new feature
-* **fix**: A bug fix
-* **docs**: Documentation only changes
-* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
-  semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug or adds a feature
-* **test**: Adding missing tests
-* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
-  generation
+Also, avoid using the commit guidelines to name your PR, for example:
 
-### Scope
+Bad
 
-The scope could be anything specifying place of the commit change. For example `app`,
-`gen`, `docs`, `gen:view`, `gen:route`, `gen:service`, etc... It is preferred to use ascope matching an available label for issues. If no label match the scope of your PR, please, get in touch with us.
+> feat(private): implement private data method
 
-### Subject
+Good
 
-The subject contains succinct description of the change:
+> Feature private information
 
-* use the imperative, present tense: "change" not "changed" nor "changes"
-* don't capitalize first letter
-* no dot (.) at the end
+In case your contribution has to do with reports, remember those are created in the develop branch, nor master or PR's.
 
-### Body
+* If we suggest changes then:
 
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes"
-The body should include the motivation for the change and contrast this with previous behavior.
+  * Make the required updates.
 
-### Footer
+  * Rebase your branch and force push to your GitHub repository (this will update your Pull Request):
 
-The footer should contain any information about **Breaking Changes** and is also the place to
-reference GitHub issues that this commit **Closes**.
+    **That's it! :tada:  Thank you for your contribution!**
 
-A detailed explanation can be found in this [document][commit-message-format].
+#### After your pull request is merged
+
+You can safely delete your branch and pull the changes from the main (upstream) repository:
+
+* Delete the remote branch on GitHub either through the GitHub web UI or your local shell as you prefer.
+
+## Coding Rules
+
+To ensure consistency throughout the source code, keep these rules in mind as you are working:
+
+* All features or bug fixes must be [tested](#tests) by one or more specs (unit-tests).
+* All methods must be documented.
+* This project follows the [GLPI development guidelines](https://readthedocs.org/projects/glpi-developer-documentation/).
+
+## Tests
+
+* Make sure test-suite passes: `phpuinit`
```

Comment:
glpi-plugin doesn't use php-unit, it uses [atoum](http://atoum.org/)

Reply:
Ok, changing...